RegularApi.Tests/BaseIT.cs

```diff
@@ -62,7 +62,7 @@ private static IWebHostBuilder CreateWebHostBuilder()
                 .ConfigureServices(services =>
                  {
                      services.AddSingleton<IMongoClient>(MongoClient);
-                     services.AddSingleton<IDaoFixture>(new DaoFixture());
+                     services.AddSingleton<DaoFixture, DaoFixture>();
```

Comment:
I know that I named this but I guess that it's a bad name, maybe you can change `DaoFixture` by something more specific, like `ApplicationFixture` or a more broad name that can involved another objects there